db/migrate/20230429100703_create_auto_tagger_bot.rb

```diff
@@ -0,0 +1,37 @@
+class CreateAutoTaggerBot < ActiveRecord::Migration[5.2]
```

Comment:
**issue** I think it's most appropriate for things in `db/migrate` as being modifications to the database schema. here are a few reasons for that:
1. at least in theory (I know this would be difficult for us in practice), we should be able to squash our migrations into one at any point, since they'll take longer and longer to run over time
2. our models can change over time, breaking past migrations when they involve them. it's a little weird to go back and change history by modifying migration files.
3. this isn't really an issue since we shouldn't be relying on database state for tests, but just wanted to point out since it can be confusing - data in migrations that is required for the application to function won't be available in tests because tests load the schema directly and therefore don't run migrations

here is a thoughtbot article that talks about some of the stuff above at more length: https://thoughtbot.com/blog/data-migrations-in-rails

**suggestion / question** for that reason, I think it would be good for us to move this to somewhere outside of migrations. I think that `lib/tasks/migrate` is intended to be for this sort of data task, but I'm not totally sure since it's not tied specifically to a database migration. @caiosba do you have any thoughts here?

**thought** also, for development, I think we should also set up all required data to run the application (like bots) outside of migrations. seeds are good for this for required things, and the thoughtbot article above suggests adding a supplemental task that runs any needed data tasks for development

Reply:
Yes, recently we've been putting those changes to data (not schema) in `lib/tasks/migrate`, but we don't run them automatically - we need to remember to run them as post-deployment tasks, manually. I still think it makes sense to put those in `lib/tasks/migrate`... we can improve the process later.

Reply:
Re-implemented this as a rake in lib/tasks/migrate